vscode/src/debugger/session.ts

```diff
@@ -316,7 +317,12 @@ export class QscDebugSession extends LoggingDebugSession {
     }
 
     if (this.config.showCircuit) {
-      await this.showCircuit();
+      const e = error as any;
```

Comment:
It's `error` already an any going by line 312? Do you need to declare e and cast here? Or can just use `error` directly.

Reply:
No it's `unknown` 😠  . This code is kinda gross in other ways too, I'll clean it up.